with-firebase-storage-signed-url-upload/App.jsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import * as ImagePicker from "expo-image-picker";
+import {getApps, initializeApp} from "firebase/app";
+import {getDownloadURL, getStorage, ref} from "firebase/storage";
+import {getFunctions, httpsCallable} from "firebase/functions";
+import {getAuth, signInAnonymously} from "firebase/auth";
+import * as FileSystem from "expo-file-system";
+import * as mime from 'react-native-mime-types';
```

Comment:
Generally we'd like to keep the examples as concise and focused on a single external package, this feature set seems better suited for a guide / tutorial since many important things are happening.